quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,149 @@
+# Quickstart Guide
+
+:information_source: Read this document before you start working on the [quickstart template](quickstart-template.md.
```

Comment:
We have used the memo and information emojis as suggested by Cameron in this thread. At the moment, we are open to changing it to curly braces to maintain consistency with the tutorial template. We can discuss this change in the next monthly meeting and include it in our style guide.
https://github.com/thegooddocsproject/templates/pull/232#discussion_r757730732

Reply:
We can probably leave it in for now. Just be aware there will be some reconciliation of templates later because your template is the only one using this standard so far. We'll just need to make a style decision down the road. :)